### PR TITLE
Explains basic differences between collaborator and membership features

### LINKED ISSUE #528: Collaborator article
We are missing an article that explains the Collaborator feature, and the difference between a domain collaborator and adding multiple users to an account.

Points to include:

- Difference between collaborator and account member
- Limitations of a collaborator
- Deprecation notice (and encourage to move to account membership)

### PR DESCRIPTION
**Pull request by kayla-broussard**
Closes #528 

Adds an article explaining the differences between account collaborators and account members, and the features/functions of both. Also mentions that the collaborator feature is deprecated.

@weppos the article as it is is fairly basic, please let me know if there are any additional mentions of functionality or edits that should be made 👍

### CODE REVIEW

File: content/articles/account-collaborator.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Previously, account access could be granted using the Collaborator feature. The Collaborator feature only provides access to DNS management. It does not allow domain management, account management or any other purchase action.
+
+
+## Account Membership
+
+<info>
+To have multiple users on one account, you must be [subscribed to an eligible plan](https://dnsimple.com/pricing).
+</info>
+
+When you add a user to an account, they will have full access rights to every resource attached to the account, including domains, contacts, SSL certificates, etc.
```
Comment (weppos) [inline content/articles/account-collaborator.markdown:35]:
```suggestion
You can [attach multiple users](articles/account-users/) to a single account, each user with individual credentials and security settings. When you add a user to an account, they will have full access rights to every resource attached to the account, including domains, contacts, SSL certificates, etc.
```

File: content/articles/account-collaborator.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+All members will also have full-admin access to the account, including the ability to access and change billing information and account details. The user will retain their login credentials and two-factor authentication configuration.
+
+You can find out more information about how to add an Account Member [here](https://support.dnsimple.com/articles/account-users/).
```
Comment (weppos) [inline content/articles/account-collaborator.markdown:39]: I moved the link in the opening, to settle the context. FYI please use relative links when cross-linking articles.

File: content/articles/account-collaborator.markdown [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+title: Differences between Account Collaborator and Account Membership
+excerpt: The differences between an Account Collaborator and an Account Member.
+categories:
+- Account
+---
+
+# Account Collaborators and Account Memberships
```
Comment (weppos) [inline content/articles/account-collaborator.markdown:8]: We rarely refers to "Account Membership". Not even in https://support.dnsimple.com/articles/account-users/.

Should we perhaps use:

> Account Collaborators and Multi-User Accounts

Comment (kayla-broussard) [reply content/articles/account-collaborator.markdown:8]: That makes perfect sense to me, I'll change 👍

File: content/articles/account-collaborator.markdown (hunk 2 above)
Comment (kayla-broussard) [reply content/articles/account-collaborator.markdown:39]: Thanks Simone - I'll do that in future!

File: content/articles/account-collaborator.markdown (hunk 3 above)
Comment (kayla-broussard) [reply content/articles/account-collaborator.markdown]: This is now fixed.

**Pull request merged**